### ISSUE TITLE
No library found for miRNA annotation in zebrafish

### ISSUE THREAD

**Issue by stefaniehoho** [mhalushka/miRge#29]
Hi,

I tried to use miRge2.0 to annotate miRNAs for zebrafish and came across the following error.

My miRge version is 2.0.6.

> Traceback (most recent call last):
>   File "/LabShares/Tools/mirge/bin/miRge2.0", line 11, in <module>
>     load_entry_point('mirge==2.0', 'console_scripts', 'miRge2.0')()
>   File "/LabShares/Tools/mirge/lib/python2.7/site-packages/mirge/__main__.py", line 116, in main
>     with open(mergeLibFile, 'r') as inf:
> IOError: [Errno 2] No such file or directory: '/LabShares/Tools/mirge/zebrafish.Libs/zebrafish/annotation.Libs/zebrafish_merges_miRBase.csv'

I found that someone had a similar issue before [https://github.com/mhalushka/miRge/issues/7](url) I downloaded the library yesterday but the library date after extraction showed May 9, 2018, which is before the latest library you suggested in that thread.

I'm wondering if you could give me some instructions to fix this.

Thanks,
Yixuan

**Comment by mhalushka** [comment]
Yixuan,
    Hi.  Sorry about that. We are in the middle of two transitions - the online storage site we are using has changed and we are updating miRge to 3.0 shortly.  Here is a link to the zebrafish library. 
https://livejohnshopkins-my.sharepoint.com/:u:/g/personal/mhalush1_jh_edu/EZ88aXFvLthRX6b-TaawAPoB1wxSV0cLL-EOpG9HsmQlSw?e=cwvYja

That file should be in here.  If you have already downloaded this file and unpacked it then it is possible that file just isn't in the correct place in your directory.  I did check that the file is in the download above.

Marc

**Comment by stefaniehoho** [comment]
Hi Marc,

Thank you so much. I have tried this library and it works for me.
I also realized a mistake in my command specifying the library directory. I provided the full directory to the library folder but the program was looking for a subdirectory of zebrafish from it. Everything works fine after correcting the command.

So glad to know miRge3.0 is coming soon. I'm looking forward to it.

Best,
Yixuan
